### DISCUSSION TITLE
Detective mode?

### DISCUSSION THREAD

**Discussion by lunarythia** [tongwentang/tongwentang-extension#67] [Q&A] [answered]
In the "auto convert" setting, what is the difference between "Simplified to Traditional" and "Detective Simplified to Traditional"?

**Comment by t7yang** [comment, marked as answer]
https://github.com/tongwentang/tongwentang-extension/blob/master/docs/preferences/preferences.md
we use the browser API to detect the web page content, and convert the content only if the content match the target to convert.
